### PR TITLE
fix: merge generated config into runtime config

### LINKED ISSUE #3558: Bug: JS runtime has issue with luis application ID
When creating a LUIS-enabled bot in Composer using the JS runtime, the bot runtime will throw this error whenever it consults LUIS:

`"Invalid `applicationId` value detected: undefined\nPlease make sure your applicationId is a valid LUIS Application Id, e.g. \"b31aeaf3-3511-495b-a07f-571fc873214b\".",`

Screen Shot 2021-04-12 at 11 29 54 AM

Screen Shot 2021-04-12 at 11 30 18 AM

### PR DESCRIPTION
**Pull request by joshgummersall**
Fixes #3558

### CODE REVIEW

**Comment by joshgummersall** [comment]
This maps to https://github.com/microsoft/botbuilder-dotnet/blob/main/libraries/Microsoft.Bot.Builder.Dialogs.Adaptive.Runtime/Extensions/ConfigurationBuilderExtensions.cs#L107 which was omitted initially.

File: libraries/botbuilder-dialogs-adaptive-runtime/src/index.ts [hunk 1]
```diff
@@ -351,6 +351,34 @@ async function addSettingsBotComponents(services: ServiceCollection, configurati
     }
 }
 
+async function addComposerConfiguration(configuration: Configuration): Promise<void> {
```
Comment (stevengum) [inline libraries/botbuilder-dialogs-adaptive-runtime/src/index.ts:354]: This can be synchronous, is it intentionally async?

**Review by stevengum — COMMENTED** [review]
One small comment, and can you add a test for this change?

File: libraries/botbuilder-dialogs-adaptive-runtime/src/index.ts (hunk 1 above)
Comment (joshgummersall) [reply libraries/botbuilder-dialogs-adaptive-runtime/src/index.ts:354]: Nope - refactored something out that was deemed unnecessary.

**Comment by joshgummersall** [comment]
@stevengum added

**Review by stevengum — APPROVED** [review]
:shipit:

**Pull request merged**